autoload/lsp/omni.vim

```diff
@@ -329,12 +336,12 @@ function! lsp#omni#get_vim_completion_items(options) abort
             \ 'icase': 1,
             \ }
         if has_key(l:completion_item, 'textEdit') && type(l:completion_item['textEdit']) == s:t_dict && has_key(l:completion_item['textEdit'], 'range') && has_key(l:completion_item['textEdit'], 'newText')
-            let l:vim_complete_item['word'] = l:completion_item['textEdit']['newText']
-            let l:start_character = min([l:completion_item['textEdit']['range']['start']['character'], l:start_character])
+            let l:character = l:completion_item['textEdit']['range']['start']['character']
+            let l:vim_complete_item['word'] = getline('.')[l:start_character:l:character - 1] . l:completion_item['textEdit']['newText']
```

Comment:
Could you create new variable for `getline('.')`?

Reply:
Done